server/controllers/users.js

```diff
@@ -150,6 +150,41 @@ class UserController {
       next(error);
     }
   }
+
+  /**
+  * Get users and their corresponding profiles
+  * @async
+  * @param {object} req - Request object
+  * @param {object} res - Response object
+  * @param {object} next The next middleware
+  * @return {json} Returns json object
+  * @static
+  */
+  static async getUsers(req, res, next) {
+    try {
+      const users = await User.findAll({
+        attributes: { exclude: ['createdAt', 'updatedAt', 'password', 'email', 'middlename'] },
```

Comment:
This would be the 3rd time I'm leaving this feedback

I don't want an exclude

use the attributes field as inclusive (this way, it excludes the fields you have not specified).


```javascript
...
attributes: [ 'field 1', 'field 2', 'field 3' ]
...
```

what you have, if the `User` model is modified and more sensitive field are added, you are exposing those fields to the world.

Please always consult documentation of libraries you are making use of.

Reply:
On it now

Reply:
I have reviewed the docs as you recommended, and I studied the use of `include` like you requested before implementing this new feedback.